core/templates/dev/head/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -167,6 +169,14 @@ oppia.factory('ExplorationObjectFactory', [
       return this.states.getAllVoiceoverLanguageCodes();
     };
 
+    Exploration.prototype.getImageCounter = function() {
+      return this.imageCounter;
```

Comment:
Make sure to add a test for this one. (I see you have defined image_counter above and accessing imageCounter here, I was expecting this to be caught in tests.)

Reply:
is it resolved?

Reply:
I have removed this function since there is no use of it.
I am extracting image counter and generating image id with a different approach. PTAL.
And sorry for not resolving.